### ISSUE TITLE
Error Prone doesn't work with Lombok 1.18.4

### ISSUE THREAD

**Issue by kamilgregorczyk** [google/error-prone#1214]
### Description of the problem / feature request:

Hi, I have a class which uses Lombok and it doesn't seem to work together with error prone plugin.
https://github.com/kamilgregorczyk/event-sourced-bank/blob/errorprone/src/main/java/com/kgregorczyk/bank/aggregates/events/MoneyTransferredEvent.java

After adding error prone plugin to maven I'm receiving exceptions related to not found getters which are generated by lombok, here's my pull request:
https://github.com/kamilgregorczyk/event-sourced-bank/pull/1/commits/7a12799ae5e21b7eab382e95830142e262266fba#diff-600376dffeb79835ede4a0b285078036

Here's some part of the error:
```
[ERROR]   location: variable event of type com.kgregorczyk.bank.aggregates.events.MoneyTransferredEvent
[ERROR] /Users/kgregorczyk/IdeaProjects/bank/src/main/java/com/kgregorczyk/bank/aggregates/EventManager.java:[82,18] cannot find symbol
[ERROR]   symbol:   method getValue()
[ERROR]   location: variable event of type com.kgregorczyk.bank.aggregates.events.MoneyTransferredEvent
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```
Here's the full error: https://pastebin.com/ua4Wu6sP

### What version of Error Prone are you using?

2.3.2

In order to reproduce it do:
```
git clone --single-branch --branch errorprone git@github.com:kamilgregorczyk/event-sourced-bank.git

cd event-sourced-bank

mvn clean compile
```

**Comment by tbroyer** [comment]
Lombok needs to be in the processor path. The processor path defaults to the compile classpath, so adding Lombok as a dependency is generally enough. But Error Prone is a JavaC Plugin and needs to be in an _explicit_ processor path, so you no longer can rely on the fallback to the compile classpath.

TL;DR: add Lombok to the `annotationProcessorPaths`.

**Comment by kamilgregorczyk** [comment]
Thanks!
